android/src/main/java/com/thebluealliance/androidclient/activities/HomeActivity.java

```diff
@@ -55,7 +55,8 @@
 
     private static final String MAIN_FRAGMENT_TAG = "mainFragment";
 
-    @Inject TBAStatusController mStatusController;
+    @Inject
+    TBAStatusController mStatusController;
```

Comment:
Keep this on one line, per Android Style guide.

You can import it into Android Studio with the instructions [here](https://github.com/the-blue-alliance/the-blue-alliance-android#code-style)